hack/install-requirements.sh

```diff
@@ -9,7 +9,7 @@ PROJECT_ROOT="${CURRENT_DIR}"/..
 
 echo "> Install Go packages/binaries"
 
-curl -sfL "https://install.goreleaser.com/github.com/golangci/golangci-lint.sh" | sh -s -- -b $(go env GOPATH)/bin v1.32.2
+curl -sfL "https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh" | sh -s -- -b $(go env GOPATH)/bin  v1.49.0
```

Comment:
What prompted this change? It doesn't seem to be relevant to this PR.

Reply:
yes, but the one who changed the linter settings never updated this. And I had to update the linter to get the same messages, which were provided in the lintercheck of this PR.

Reply:
Gotcha 👍